test/domain_bridge/test_domain_bridge_end_to_end.cpp

```diff
@@ -252,3 +253,25 @@ TEST_F(TestDomainBridgeEndToEnd, decompress_mode)
   bridge.add_to_executor(spinner.get_executor());
   EXPECT_TRUE(poll_condition([&got_message]() {return got_message.load();}, 3s));
 }
+
+TEST_F(TestDomainBridgeEndToEnd, create_bidirectional_bridge)
+{
+  const std::string topic_name("test_bidirectional");
+
+  // Create a publisher on domain 1
+  auto pub = node_1_->create_publisher<test_msgs::msg::BasicTypes>(topic_name, 1);
+
+  // Bridge the publisher topic bidirectionally
+  domain_bridge::TopicBridgeOptions topic_bridge_options;
+  topic_bridge_options.bidirectional(true);
+  domain_bridge::DomainBridgeConfig config;
+  config.topics.push_back(
+  {
+    {topic_name, "test_msgs/msg/BasicTypes", kDomain2, kDomain1},
+    topic_bridge_options
+  });
+  domain_bridge::DomainBridge bridge(config);
+
+  // 'to' domain is 1, but since we are bridging both ways, publisher should also appear on domain 2
+  ASSERT_TRUE(wait_for_publisher(node_2_, topic_name));
```

Comment:
I would test both directions, just in case

```suggestion
  ASSERT_TRUE(wait_for_publisher(node_1_, topic_name));
  ASSERT_TRUE(wait_for_publisher(node_2_, topic_name));
```

Reply:
Thanks, done in 90e7647.